website/pages/en/sunrise.mdx

```diff
@@ -1,233 +1,79 @@
 ---
-title: Sunrise + Upgrading to The Graph Network FAQ
+title: Post-Sunrise + Upgrading to The Graph Network FAQ
 ---
 
-> Note: This document is continually updated to ensure the most accurate and helpful information is provided. New questions and answers are added on a regular basis. If you can’t find the information you’re looking for, or if you require immediate assistance, [reach out on Discord](https://discord.gg/graphprotocol). If you are looking for billing information, then please refer to [billing](/billing/).
+> Note: The Sunrise of Decentralization ended June 12th, 2024.
 
-## What is the Sunrise of Decentralized Data?
+## What was the Sunrise of Decentralized Data?
 
-The Sunrise of Decentralized Data is an initiative spearheaded by Edge & Node. The goal is to enable subgraph developers to seamlessly upgrade to The Graph’s decentralized network.
+The Sunrise of Decentralized Data was an initiative spearheaded by Edge & Node. This initiative enabled subgraph developers to upgrade to The Graph’s decentralized network seamlessly.
 
-This plan draws on many previous developments from The Graph ecosystem, including an upgrade Indexer to serve queries on newly published subgraphs, and the ability to integrate new blockchain networks to The Graph.
+This plan drew on previous developments from The Graph ecosystem, including an upgraded Indexer to serve queries on newly published subgraphs.
 
-### What are the phases of the Sunrise?
+### What happened to the hosted service?
 
-**Sunray**: Enable support for hosted service chains, introduce a seamless upgrade flow, offer a free plan on The Graph Network, and provide simple payment options.  
-**Sunbeam**: The upgrade window that subgraph developers will have to upgrade their hosted service subgraphs to The Graph Network. This window will end at 10 a.m. PT on June 12th 2024.  
-**Sunrise**: Hosted service endpoints will no longer be available after 10 a.m. PT on June 12th, 2024.
+The hosted service query endpoints are no longer available, and developers cannot deploy new subgraph versions on the hosted service.
```

Comment:
Not just new versions, but new subgraphs too, right?

Reply:
Both! So I'll just state it as "subgraphs"